### PR TITLE
List transformations sample

### PR DESCRIPTION
**Pull request by mhdostal**
The List transformations sample

### CODE REVIEW

File: arcgis-runtime-samples-macos/Geometry/List transformations/ListTransformationsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,190 @@
+//
+//  ListTransformationsViewController.swift
```
Comment (philium) [inline arcgis-runtime-samples-macos/Geometry/List transformations/ListTransformationsViewController.swift]: Please replace with the standard Esri source code header.

**Comment by mhdostal** [comment]
@philium I updated the copyright and merged in the latest release/100.3 branch.  Please re-review.

**Comment by mhdostal** [comment]
@njarecha Do you want to review this?

File: arcgis-runtime-samples-macos/Geometry/List transformations/ListTransformationsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+import Cocoa
+import ArcGIS
+
+class ListTransformationsViewController: NSViewController, NSTableViewDataSource, NSTableViewDelegate {
+
+    @IBOutlet var tableView: NSTableView!
+    @IBOutlet var mapView: AGSMapView!
+    @IBOutlet var orderByMapExtent: NSButton!
+    
+    var datatumTransformations = [AGSDatumTransformation]()
```
Comment (njarecha) [inline arcgis-runtime-samples-macos/Geometry/List transformations/ListTransformationsViewController.swift]: Rename datatumTransformations -> datumTransformations

File: arcgis-runtime-samples-macos/Geometry/List transformations/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## How to use the sample
+Optionally, begin by adding projection engine data to your device before running this sample. When you use the projection engine data, more transformations are available for use.
+
+Tap on a listed transformation to reproject the point geometry (shown in red) using the selected transformation. The reprojected geometry will be shown in blue. If there are grid-based transformations for which projection engine files are not available on your device, these will be listed in an items details. The default transformation is highlighted in the list.
+
+## How it works
+The sample sets the location of projection engine data on the device by calling `AGSTransformationCatalog.setProjectionEngineDirectory`.
+
+The list of `AGSDatumTransformations` is created by calling `AGSTransformationCatalog.transformationsBySuitability`, passing in the `AGSSpatialReference` of the original geometry (the input spatial reference) and that of the `AGSMap` (the output spatial reference). Depending on the state of a switch control, the current visible extent of the map is used to sort the list by suitability.
```
Comment (njarecha) [inline arcgis-runtime-samples-macos/Geometry/List transformations/README.md]: `AGSDatumTransformations` should be `AGSDatumTransformation`

**Comment by mhdostal** [comment]
@njarecha Done, please re-review.

**Pull request merged**